tool/site/client-react/src/components/prescription-summary/prescriptionLevelDetails.tsx

```diff
@@ -0,0 +1,135 @@
+import {SummaryList} from "nhsuk-react-components"
+import * as React from "react"
+import {CommunicationRequest, CommunicationRequestPayload, MedicationRequest} from "fhir/r4"
+import {formatDate} from "../../formatters/dates"
+import {
+  getNumberOfRepeatsAllowedExtension,
+  getNumberOfRepeatsIssuedExtension, getPerformerSiteTypeExtension,
+  getRepeatInformationExtension
+} from "../../fhir/customExtensions"
+
+export function createPrescriptionLevelDetails(medicationRequest: MedicationRequest, communicationRequests?: Array<CommunicationRequest>): PrescriptionLevelDetailsProps {
+  const prescriptionId = medicationRequest.groupIdentifier.value
+
+  const repeatExtension = getRepeatInformationExtension(medicationRequest.extension)
+  const repeatsIssuedExtension = getNumberOfRepeatsIssuedExtension(repeatExtension.extension)
+  const repeatIssued = repeatsIssuedExtension.valueInteger
+  const repeatsAllowedExtension = getNumberOfRepeatsAllowedExtension(repeatExtension.extension)
+  const repeatAllowed = repeatsAllowedExtension.valueInteger
+
+  const authoredOn = formatDate()
+  const startDate = medicationRequest.dispenseRequest.validityPeriod?.start ?? new Date().toISOString().slice(0, 10)
```

Comment:
Could use `formatDate(medicationRequest.dispenseRequest.validityPeriod?.start)` and then we'll use a consistent date format for authoredOn and startDate.